docs/source/development.rst

```diff
@@ -131,6 +131,73 @@ If this doesn't work, you can skip specific checks by prepending ``SKIP=<hook-id
 We welcome your contributions even if your workflow differs from what we recommend here.
 
 
+Updating Dependencies
+---------------------
+
+There are three categories of dependencies that can be found in this repository:
+*Github Actions*, *Poetry* and *pre-commit hooks*.
+Currently there is no programmatic way to update the github actions we depend on.
+One has to manually update the commit hashes used.
+
+Poetry dependencies
+```````````````````
+
+Poetry differentiates between direct and indirect dependencies.
+Direct dependencies are specified in ``pyproject.toml`` -- usually with a fixed version.
+Updating these can introduce bugs through breaking changes in the API exposed by
+these dependencies. That's why we'll focus on updating indirect dependencies first.
+Running the following and committing the resulting changes to ``poetry.lock`` will
+do the trick provided that set up *pre-commit* according to this guide.
+
+
+.. code:: bash
+
+   # 1. Update indirect dependencies (stored in poetry.lock)
+   poetry update
+
+   # 2. Stage and commit (runs pre-commit hooks)
+
+
+Direct dependencies are updated by changing the version specified in ``pyproject.toml``.
+
+
+.. code:: bash
+
+   # 1. Determine latest version of a package
+   pip show packaging
```

Comment:
The [`pip show`](https://pip.pypa.io/en/stable/cli/pip_show/) command displays whatever version happens to be installed locally, not necessarily the latest available version. We'll have to look up the latest version on pypi.

Reply:
Oh right